Utilities/StorageFactory/src/StatisticsSenderService.cc

```diff
@@ -383,6 +384,7 @@ void StatisticsSenderService::fillUDP(const std::string &siteName,
   if (!siteName.empty()) {
     os << "\"site_name\":\"" << siteName << "\", ";
   }
+  os << "\"cmssw_version\":" << edm::getReleaseVersion() << ", ";
```

Comment:
@makortel , `edm::getReleaseVersion() returns string, so shouldn't this be enclosed in double quotes?

Reply:
It turned out to return a string that includes quotation marks.

Reply:
(maybe I should add a comment there since I didn't expect that either)

Reply:
ah ok